theano/tensor/nnet/corr.py

```diff
@@ -73,7 +74,11 @@ def c_libraries(self):
         return ldflags()
 
     def c_compile_args(self):
-        return ldflags(libs=False, flags=True)
+        ret = ldflags(libs=False, flags=True)
+        # Add the -fopenmp flags
+        ret += super(BaseCorrMM, self).c_compile_args()
+
+        return ret
```

Comment:
The method c_headers must also be updated to get the missing include added by the parent.